JKLT-org/EtherView
# database setup and preliminary authentication

##Overview
**Issue Type**
[] Bug
[X] Feature
[]Tech debt

**Description**
Implemented authentication, database and got server file up and running on port 3000

**Steps to Reproduce Bug/Validate Feature/Confirm Tech Debt Fix**
see files under server folder for database connection, server setup, and authentication logic - test with postman

**Previous Behavior**
features did not exist

**Expected Behavior**
accurate signup and login 

**How Has This Been Tested**
test data for users has been tested through postman and has successfully created users and been able to login

**Screenshots & Videos**
N/A 

**Additional Context**
N/A
